os/net/mac/tsch/tsch-schedule.c

```diff
@@ -224,9 +224,6 @@ tsch_schedule_add_link(struct tsch_slotframe *slotframe,
     if(timeslot > (slotframe->size.val - 1)) {
       LOG_ERR("! add_link invalid timeslot: %u\n", timeslot);
       return NULL;
-    } else if(channel_offset > 15) {
-      LOG_ERR("! add_link invalid channel_offset: %u\n", channel_offset);
-      return NULL;
```

Comment:
I would like it if we could keep this check

Reply:
I wonder why? There can be hopping sequences with more than 16 elements. I don't think standard says anything about this. Plus, many of the current hopping sequences have less than 16 elements, including the default one! As you know, with hopping sequence 4 elements long, the channel offsets 1, 5, 9 and 13 are all going to map to the same physical channel - so why not allow channel offsets 17, 21 etc. as well?

Reply:
well yes you're right; I had been somehow assuming that the standard mandated this but doesn't seem to be any other constraint than it being 16bit unsigned :)